lldb/include/lldb/ValueObject/DILAST.h

```diff
@@ -88,6 +89,29 @@ class IdentifierNode : public ASTNode {
   std::string m_name;
 };
 
+class MemberOfNode : public ASTNode {
+public:
+  MemberOfNode(uint32_t location, ASTNodeUP base, bool is_arrow,
+               ConstString name)
+      : ASTNode(location, NodeKind::eMemberOfNode), m_base(std::move(base)),
+        m_is_arrow(is_arrow), m_field_name(name) { }
+
+  llvm::Expected<lldb::ValueObjectSP> Accept(Visitor *v) const override;
+
+  ASTNode *base() const { return m_base.get(); }
+  bool IsArrow() const { return m_is_arrow; }
```

Comment:
Let's be consistent with the names here. I see we've used `GetFoo` elsewhere.

@kuilpd, I missed this in the UnaryOpNode (because it was self-consistent at least, but I think that one should also use the GetFoo scheme.

Reply:
Done.